src/Console/Commands/HashApiKeys.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Givebutter\LaravelKeyable\Console\Commands;
+
+use Givebutter\LaravelKeyable\Models\ApiKey;
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Facades\DB;
+
+class HashApiKeys extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'api-key:hash {--id= : ID of the API key you want to hash}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Hash existing API keys';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        DB::transaction(function () {
+            ApiKey::query()
+                ->withTrashed()
+                ->when($this->option('id'), function (Builder $query, int $id) {
+                    $query->where('id', $id);
+                })
+                ->eachById(function (ApiKey $apiKey) {
+                    $apiKey->update([
+                        'key' => hash('sha256', $apiKey->key),
+                    ]);
```

Comment:
It might be a good idea to add a where clause to only update keys that don't already look like `sha256` hashes, just in case the command is run more than once. In fact, if the `--id` argument is used first and then again without, that key would be double hashed.

I think `->where('key', 'not regexp', '^[0-9a-f]{64}$')` should work.

Reply:
SQlite doesn't have support for `regexp`, so the tests were failing, had to replace it with `->whereRaw('LENGTH(key) != 64')`, which should give us the same result, I added a test.